controller_manager/test/test_hardware_resources/test_actuator_hardware.cpp

```diff
@@ -0,0 +1,62 @@
+// Copyright 2020 ros2_control Development Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+
+#include "./test_actuator_hardware.hpp"
```

Comment:
Can you fill me in on this one please? Is it to enforce locality of the include or a mishap from your IDE?

Why not use ` #include "test_actuator_hardware.hpp"` ?

Reply:
cpplint expects `include`s with a directory

Reply:
I should have guessed, most strange formatting originates from a linter